daprdocs/content/en/developing-applications/building-blocks/service-invocation/service-invocation-overview.md

```diff
@@ -38,13 +38,29 @@ The diagram below is an overview of how Dapr's service invocation works.
 6. Dapr forwards the response to Service A's Dapr sidecar.
 7. Service A receives the response.
 
+The diagram below is an overview of how Dapr's service invocation works when invoking non-Daprized endpoints.
```

Comment:
I moved everything to the other file like you recommended, but my thinking is do we want to include inconsistency in diagrams by me duplicating my diagram for non-Dapr endpoint invocation without the numbers for the overview page? I'm not the biggest fan of inconsistencies like that. I like the change to just a paragraph here, but if they click to then look for more information on non-Dapr endpoint invocation, then would it be sufficient to just have the diagram there, instead of duplicating without numbers on the service invocation overview page?